jestconfig.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
```

Comment:
We should rename this to jest.config.js so it's automatically picked up by the jest cli